### PR TITLE
improving docs / README.md refactoring

### PR DESCRIPTION
**Pull request by mheese**
Check out the branch directly to get the rendered view of the new README: https://github.com/aporeto-inc/trireme-kubernetes/tree/improve-docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,41 +7,61 @@
...
 
-* [Trireme-CSR](https://github.com/aporeto-inc/trireme-csr): an identity service that is used in order to automatically generate certificates and asymmetric keypairs for each Trireme-Kubernetes instance
+* [Trireme-Statistics](https://github.com/aporeto-inc/trireme-statistics) (optional): the monitoring and statistics bundle that currently implements the trireme-lib collector interface for InfluxDB. Flows and Container events can be displayed in either Grafana, Chronograf or Trireme-Graph - which shows a generated graph specifically for Kubernetes network flows between pods. Depending on your use-case, some or all of those frontend tools can be deployed.
 
-* [Trireme-Statistics](https://github.com/aporeto-inc/trireme-statistics) bundle: the monitoring and statistics bundle that relies on InfluxDB. Flows and Container events can be displayed in either Grafana, Chronograf or a generated graph specifically for Kubernetes flows. Depending on your use-case, some or all of those frontend tools can be deployed.
 
-## Getting started with Trireme-Kubernetes
+## Prerequisites
```
Comment (bvandewalle) [inline README.md]: This should be in the end IMO.

File: README.md [hunk 2]
```diff
@@ -7,41 +7,61 @@
 
 ----
 
-Trireme-Kubernetes is a simple, straightforward implementation of the _Kubernetes Network Policies_ specifications. Yet, it is independent from the used networking backend and works in any Kubernetes cluster.
+TL;DR? Jump to the **[Getting Started](#getting-started)** section.
+
+Trireme-Kubernetes is a simple, straightforward implementation of the _Kubernetes Network Policies_ specifications. It is independent from the used networking backend and works in _any_ Kubernetes cluster - even in managed Kubernetes clusters like [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs/) or [Azure Container Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/).
+
+One of its powerful features is that you can deploy it to _multiple_ Kubernetes clusters and secure network traffic between specific pods of the different clusters (to secure e.g. MySQL replication or a MongoDB replicaset). You can furthermore integrate it with any other _non-containerized_ Linux service/process or any other docker container using [Trireme-Example](https://github.com/aporeto-inc/trireme-example).
```
Comment (bvandewalle) [inline README.md]: The first sentence is very good, but the second one is a bit confusing. We should rework it to specifically target Kubernetes and explain that it is then using trireme-lib which can be used accross processes and Dockers

File: README.md [hunk 3]
```diff
@@ -7,41 +7,61 @@
...
+
+![Trireme-Kubernetes-Architecture](docs/trireme-kubernetes-architecture.png)
+
+Trireme-Kubernetes consists of several components - not all of them are required:
 
-![Kubernetes-Trireme ecosystem](docs/architecture.png)
+* [Trireme-Kubernetes](https://github.com/aporeto-inc/trireme-kubernetes): the enforcement service which polices network connections (a.k.a "flows" in Trireme terminology) based on standard `NetworkPolicies` defined on the Kubernetes API
 
-* Trireme-Kubernetes: the enforcement service which polices flows based on standard NetworkPolicies defined on the Kubernetes API
+* [Trireme-CSR](https://github.com/aporeto-inc/trireme-csr): an identity service (basically a CA) that is used to automatically sign certificates and generate asymmetric KeyPairs for each Trireme-Kubernetes instance.
```
Comment (bvandewalle) [inline README.md]: I would stress that Trireme-CSR and Trireme-Statistics are both optional. We can actually build a quick start guide with PSK.

File: README.md (hunk 2 above)
Comment (mheese) [reply README.md]: agreed, let me work on this some more

File: README.md (hunk 1 above)
Comment (mheese) [reply README.md]: hm, I'd argue that it is quite normal to have the requirements section before the installation section

**Comment by mheese** [comment]
This PR is directly related to #136 .

**Comment by bvandewalle** [comment]
LGTM Thanks @mheese . 

I think this is in a good state, and as discussed before let's iterate :-)

**Pull request merged**